Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -342,20 +335,14 @@ theorem toQuadraticForm' [DecidableEq n] {M : Matrix n n ℝ} (hM : M.PosDef) :
 /-- The eigenvalues of a positive definite matrix are positive -/
 lemma eigenvalues_pos [DecidableEq n] {A : Matrix n n 𝕜}
     (hA : Matrix.PosDef A) (i : n) : 0 < hA.1.eigenvalues i := by
-  rw [hA.1.eigenvalues_eq, hA.1.transpose_eigenvectorMatrix_apply]
+  simp only [hA.1.eigenvalues_eq]
   exact hA.re_dotProduct_pos <| hA.1.eigenvectorBasis.orthonormal.ne_zero i
 
 theorem det_pos [DecidableEq n] {M : Matrix n n ℝ} (hM : M.PosDef) : 0 < det M := by
-  rw [hM.isHermitian.det_eq_prod_eigenvalues]
-  apply Finset.prod_pos
-  intro i _
-  rw [hM.isHermitian.eigenvalues_eq]
-  refine hM.2 _ fun h => ?_
-  have h_det : hM.isHermitian.eigenvectorMatrixᵀ.det = 0 :=
-    Matrix.det_eq_zero_of_row_eq_zero i fun j => congr_fun h j
-  simpa only [h_det, not_isUnit_zero] using
-    isUnit_det_of_invertible hM.isHermitian.eigenvectorMatrixᵀ
-#align matrix.pos_def.det_pos Matrix.PosDef.det_pos
```

Comment:
Can you put this `#align` back now?

Reply:
Yes. I didn't fix all of the #align for PosDef...I should go do that...

Reply:
Ok. I think the one you mentioned was the only one. So done...